vitessce/wrappers.py

```diff
@@ -488,6 +505,15 @@ def get_expression_matrix(base_url):
                 return obj_file_def
             return None
         return get_expression_matrix
+    
+    def auto_view_config(self, vc):
+        dataset = vc.add_dataset().add_object(self)
+        scatterplot = vc.add_view(dataset, cm.SCATTERPLOT, mapping=self._mappings_obsm[0].split('/')[-1])
+        cell_sets = vc.add_view(dataset, cm.CELL_SETS)
+        genes = vc.add_view(dataset, cm.GENES)
+        heatmap = vc.add_view(dataset, cm.HEATMAP)
+
+        vc.layout((scatterplot | (cell_sets / genes)) / heatmap)
```

Comment:
```suggestion
        spatial = vc.add_view(dataset, cm.SPATIAL)
        if self._spatial_polygon_obsm  is not None or self._spatial_centroid_obsm is not None:
            vc.layout((scatterplot | spatial) / (heatmap | (cell_sets / genes)))
        else:
            vc.layout((scatterplot | (cell_sets / genes)) / heatmap)
```
something like this modulo the indentation and I'll test this out in my PR